### PR TITLE
remove cluster-test

### PR DESCRIPTION
**Pull request by bmwill**
Now that forge has replaced cluster-test we can remove cluster-test code and docker builds now that its unused. Also enables us to remove the now unused json-log event queue which only existed to support cluster test.

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -290,22 +290,6 @@ jobs:
             echo "PREV_TAG=$compat_prev_tag" >> $GITHUB_ENV
             echo "BUILD_PREV=0" >> $GITHUB_ENV
           fi
-      - name: build extra images
```
Comment (zihaoccc) [inline .github/workflows/ci-test.yml:293]: I think we do need to keep this since compatibility test is just disabled but not removed?

Comment (bmwill) [reply .github/workflows/ci-test.yml:293]: This specifically does cti-codebuild.sh, which doesn't do forge builds correct?

Comment (zihaoccc) [reply .github/workflows/ci-test.yml:293]: we might need to change cti-codebuild script under land-blocking at line 
```
docker/build-aws.sh --build-all-cti --version $VERSION --addl_tags canary,${ADDL_TAG}
```
to 
```
docker/build-aws.sh --build-forge --version $VERSION --addl_tags canary,${ADDL_TAG}
```

I can fix this in another PR

File: consensus/src/block_storage/block_tree.rs [hunk 2]
```diff
@@ -448,12 +447,6 @@ impl BlockTree {
             committed_round = committed_round,
             block_id = block_to_commit.id(),
         );
-        event!("committed",
```
Comment (zihaoccc) [inline consensus/src/block_storage/block_tree.rs:451]: it looks good to me but I am not very clear about this part change

Comment (bmwill) [reply consensus/src/block_storage/block_tree.rs:451]: This is the special out-of-bound health check thing that CT used that we've been wanting to get rid of for a long time

File: testsuite/forge/src/github.rs [hunk 3]
```diff
@@ -49,7 +49,7 @@ impl GitHub {
         })?;
         let request = self.client.get(url);
         let response = request
-            .header(USER_AGENT, "diem-cluster-test")
+            .header(USER_AGENT, "diem-forge")
```
Comment (zihaoccc) [inline testsuite/forge/src/github.rs:52]: nice catch

**Review by zihaoccc — APPROVED** [review]

**Comment by zihaoccc** [comment]
what a big step hurray

**Pull request closed without merging**